### ISSUE TITLE
The definition of uniform distribution is different between the paper and code

### ISSUE THREAD

**Issue by maoxin7676** [introlab/odas#276]
* The eq (70 ) in "Lightweight and Optimized Sound Source Localization and Tracking Methods for Open and Closed Microphone Array Configurations" caculates the uniform distribution p = K_hat/(4*pi*K)   , where K_hat denotes the number of points scanned, and K the total number of points needed to discretize the entire sphere。
* The line 153 in mod_sst.c is diffuse_cst = 1.0f / (4.0f * M_PI * ((float) nPointsActive) / ((float) spatialindexes->nPoints)), the value of diffuse_cst is large than 1, i think it should be diffuse_cst = (((float)nPointsActive) / (4.0f * M_PI * (float)spatialindexes->nPoints)); 
* Please help confirm whether the above description is correct

**Comment by FrancoisGrondin** [comment]
Yes you are right, this seems to be a mistake in the code. It's probably absorbed by some other hand-tuned parameters found empirically, but should be changed to what you said to match the equations in the paper.
